modules/tile-converter/src/i3s-converter/types.ts

```diff
@@ -163,3 +163,16 @@ export type TypedArrayConstructor =
   | Uint32ArrayConstructor
   | Float32ArrayConstructor
   | Float64ArrayConstructor;
+
+export type Tiles3DLoadOptions = {
```

Comment:
Why are these options defined in the tile-converter?

Reply:
This is  the i3s-converter specific options

Reply:
I've found that we can use `Tiles3DLoaderOptions` here. Fixed in https://github.com/visgl/loaders.gl/pull/2520